LexiconSite/base/views.py

```diff
@@ -1,14 +1,26 @@
 from django.shortcuts import render
 from django.http import HttpResponse
 
+# TODO: Update the html file names
```

Comment:
all the file names here look correct? should there be something else we're doing?